client/ayon_core/tools/sceneinventory/model.py

```diff
@@ -321,7 +321,17 @@ def _add_containers(self, containers, parent=None):
             )
             group_node["representation"] = repre_id
             group_node["version"] = version_entity["version"]
-            group_node["highest_version"] = highest_version["version"]
+
+            # We check against `abs(version)` because we allow a hero version
+            # which is represented by a negative number to also count as
+            # latest version
+            # If a hero version for whatever reason does not match the latest
+            # positive version number, we also consider it outdated
+            group_node["isOutdated"] = (
```

Comment:
I would say that hero versions should not be considered as outdated if are not from latest version...
```python
if version_entity["version"] < 0:
    is_outdated = False
else:
    last_version = highest_version_by_product_id.get(version_entity["productId"])
    is_outdated = version_entity["version"] != last_version
group_node["isOutdated"] = is_outdated
```

Reply:
If the hero version is v001 and the latest version is v006 (so for whatever reason somehow no hero versions got created or updated anymore?) you'd still want to the v001 hero version to be counted as "latest"? Seems somewhat counter-intuitive?

I'm fine with it to make the change though, since we don't usually use hero versions on our end. But it does make me doubt whether it really makes sense that way.

Reply:
>  you'd still want to the v001 hero version to be counted as "latest"?

Not as latest, but not as outdated.

Reply:
I suspect that in OpenPype it would not be considered outdated - no. I'll refactor the logic here as you requested to remain in sync with that. 👍